<launch>
    <node name="get_dxl_position" pkg="deltarobot" type="get_dxl_position.py" />
    <node name="set_dxl_speed" pkg="deltarobot" type="set_dxl_speed.py" />
    <node name="pub_encoder_position" pkg="deltarobot" type="pub_encoder_position.py" />
    <node name="dxl_PID_controller" pkg="deltarobot" type="dxl_PID_controller.py" />
    <node name="set_end_position" pkg="deltarobot" type="set_end_position.py" />
    <node name="get_end_force" pkg="deltarobot" type="get_end_force.py" />
    <node name="get_end_position" pkg="deltarobot" type="get_end_position.py" />
</launch>